packages/nextjs/pages/index.tsx

```diff
@@ -48,6 +48,31 @@ const Home: NextPage<{
         </div>
       </div>
 
+      {/* Quote section*/}
+      <div className="bg-base-100">
+        <div className="container max-w-[90%] lg:max-w-7xl m-auto py-16 lg:py-20 lg:pl-20 flex flex-col-reverse lg:flex-row items-center lg:items-start lg:gap-10">
+          {/* Quote image */}
+          <div className="flex flex-col items-center max-w-md lg:max-w-md lg:mt-6">
+            <Image
+              src="/assets/austin-pfp.png"
+              alt="austin griffith"
+              width={120}
+              height={120}
+              className="rounded-[150px] shadow-xl"
+            />
+          </div>
+          {/* Quote Text */}
+          <div className="md:max-w-[80%] lg:max-w-[70%] flex flex-col items-center lg:items-start">
+            <p className="font-thin text-2xl lg:text-2xl text-center lg:text-left mb-10 lg:mb-4">
```

Comment:
Maybe we could use text-xl for smaller devices? Don't know if it was intended to set `text-2xl lg:text-2xl` or it's a typo and wanted to set `text-xl lg:text-2xl`

Reply:
Yeah I agree, I think I changed it to `text-2xl` cause it looked a bit small but with the computer screen is not very realistic anyway. Changing it back to `text-xl`